src/components/organisms/AppRouter/Provided.tsx

```diff
@@ -13,12 +14,18 @@ export const Provided: React.FC<ProvidedProps> = ({
   withRelatedVenues = false,
 }) => {
   const { worldSlug, spaceSlug } = useSpaceParams();
-  const { space, spaceId } = useWorldAndSpaceBySlug(worldSlug, spaceSlug);
+  // @debt This provider is used in cases where we don't have a space slug
+  // Refactoring that is more risky than makes sense right now (big events
+  // coming up). Therefore, we do best effort here. This provider shouldn't
+  // need a space at all. Related venues can be derived entirely from a world
+  // now.
+  const { world } = useWorldBySlug(worldSlug);
+  const { spaceId } = useWorldAndSpaceBySlug(worldSlug, spaceSlug);
```

Comment:
This hook returns world. I don't think you need to use `useWorldBySlug` hook here

Reply:
See the big comment :(

Reply:
Improved comment in ea34b8ac